parlai/tasks/wrapper/agents.py

```diff
@@ -23,37 +23,19 @@
 from parlai.core.teachers import create_task_agent_from_taskname, Teacher
 
 
-class LabelToTextTeacher(Teacher):
+class AbstractWrapperTeacher(Teacher):
     """
-    Teacher that will shift message['labels'][0] into message['text'] for whatever task
-    is specified with --label-to-text-task.
-
-    Because the dialogue history is effectively overwritten by this action, all episodes
-    will be flattened into one example each.
+    Abstract teacher working as base for wrapper teachers.
     """
 
-    @classmethod
-    def add_cmdline_args(cls, argparser):
-        # Teacher has no args
-        parser = argparser.add_argument_group('LabelToText args')
-        parser.add_argument(
-            '-l2tt',
-            '--label-to-text-task',
-            type=str,
-            help='The task whose labels will get shifted into the text field',
-        )
-
     def __init__(self, opt: Opt, shared=None):
-        if ',' in opt['task']:
-            raise ValueError('LabelToTextTeacher cannot be used with multiple tasks!')
-        self.id = opt['task']
-        self.opt = opt
-        if shared and 'task' in shared:
-            self.task = create_agent_from_shared(shared['task'])
-        else:
-            opt_singletask = copy.deepcopy(opt)
-            opt_singletask['task'] = opt['label_to_text_task']
-            self.task = create_task_agent_from_taskname(opt_singletask)[0]
+        pass
+
+    def act(self):
```

Comment:
You can slap a `@abstractmethod` decorator here to signify that this method needs to be defined by all subclasses :)